### ISSUE TITLE
[FEATURE REQUEST] Compact View

### ISSUE THREAD

**Issue by 0xlne** [zhanghai/MaterialFiles#236]
A kind request to enable an option for even more compact mode to display more files in a list.

**Comment by 07416** [comment]
Also filenames can't often be seen in their entirety, not at all essentially if there's any prefix (e.g. "episode#-").

**Comment by haarp** [comment]
Agreed, there is a lot of wasted space in the default view. It could easily display twice as many files at once without getting crowded.

Thanks!

**Comment by darkdragon-001** [comment]
Agree, vertival spacing could easily be reduced without looking crowded. 

Apart from this, it's the best file explorer app on Android!

**Comment by pgaskin** [comment]
At the minimum, it's be nice to be able to reduce the font size and padding of the file list to something more similar to the sidebar.

**Comment by pgaskin** [comment]
These commands will reduce the default file list height from 72dip to 56dip, which removes the extra padding at the top/bottom without making it look cluttered:

```
adb shell cmd overlay fabricate --name MaterialFilesListHeight --target me.zhanghai.android.files me.zhanghai.android.files:dimen/two_line_list_item_height 0x5 0x00003801
adb shell cmd overlay enable com.android.shell:MaterialFilesListHeight
```

See [this gist](https://gist.github.com/pgaskin/0d4d3550cbb944161dd109c21b9a7332) for more info.

**Comment by pgaskin** [comment]
@zhanghai would it be possible to add a toggle to preferences to control this?
